features/authentication.feature

```diff
@@ -95,7 +95,7 @@ Feature: OAuth authentication
       Duplicate value for "description"
       """
     And the exit status should be 1
-    And the file "../home/.config/hub" should not exist
+    And the file "../home/.config/hub" should contain ""
```

Comment:
Ah, nice catch. These tests did expect no file being created, but we create one anyway to check for writeability.

How about a workaround: `CheckWriteable()` would first test if the file is present before trying to write it. If it wasn't present, remember that. Then, after checking for writeability, and if the file wasn't present previously, delete it. How does that sound?